apps/site-landing/src/containers/Compare/FormCompare/index.tsx

```diff
@@ -26,55 +23,31 @@ const translations = defineMessages({
 
 export const FormCompare = () => {
 
-    const intl = useIntl();
-    const labelStartingValue = intl.formatMessage(translations.labelStartingValueText);
-    const labelStartingValueCurrency = intl.formatMessage(translations.labelStartingValueCurrencyText);
-    const scaleStartingType = "currency";
-    const minRangeStartingValue = 1000;
-    const maxRangeStartingValue = 100000;
-    const stepRangeStartingValue = 100;
-    const minRangeScaleStartingValue = 1000;
-    const medRangeScaleStartingValue = 5000;
-    const maxRangeScaleStartingValue = 100000;
-
-    const labelDurationValue = intl.formatMessage(translations.rangeDuration);
-    const labelDurationYear = intl.formatMessage(translations.rangeDurationYear);
-    const scaleDurationType = "unit";
-    const minRangeDurationValue = 1;
-    const maxRangeDurationValue = 80;
-    const stepRangeDurationValue = 1;
-    const minRangeScaleDurationValue = 1;
-    const medRangeScaleDurationValue = 40;
-    const maxRangeScaleDurationValue = 80;
-
-    return (
-      <div id={styles.variablesContainer}>
-
-        <RangeFieldAndScale
-            labelValue={labelStartingValue}
-            labelValueCurrency={labelStartingValueCurrency}
-            scaleType={scaleStartingType}
-            minRange={minRangeStartingValue}
-            maxRange={maxRangeStartingValue}
-            stepRange={stepRangeStartingValue}
-            minRangeScale={minRangeScaleStartingValue}
-            medRangeScale={medRangeScaleStartingValue}
-            maxRangeScale={maxRangeScaleStartingValue}/>
-
-        <RangeFieldAndScale
-            labelValue={labelDurationValue}
-            labelValueCurrency={labelDurationYear}
-            scaleType={scaleDurationType}
-            minRange={minRangeDurationValue}
-            maxRange={maxRangeDurationValue}
-            stepRange={stepRangeDurationValue}
-            minRangeScale={minRangeScaleDurationValue}
-            medRangeScale={medRangeScaleDurationValue}
-            maxRangeScale={maxRangeScaleDurationValue}/>
-
-        <Button secondary className={ `${styles.buttonContainer} x16spaceBefore` }>
-          <FormattedMessage {...translations.button} />
-        </Button>
-      </div>
-    );
+  // TODO: Connect all o' dis
+  const onChange = () => {};
+
+  return (
+    <div id={styles.variablesContainer}>
+
+      <RangeFieldAndScale
+        label={translations.startingValue}
+        scaleType="currency"
+        currency="CAD"
+        maximum={1000}
+        step={1}
+        onChange={onChange}
+      />
```

Comment:
This file isn't the topic of review.  Updated to compile, doesn't matter if it 'works'